JanePhotoBrowser/PhotoBrowser/PhotoBrowserFullscreen/PhotoBrowserFullscreenViewController.swift

```diff
@@ -107,6 +127,45 @@ public class PhotoBrowserFullscreenViewController: UIViewController {
         self.pagedView.currentImageView.addGestureRecognizer(pan)
     }
     
+    private func setupZoomButton() {
+        guard UIAccessibility.isVoiceOverRunning else { return }
+        self.view.addSubview(self.zoomButtonContainer) {
+            $0.left.pinToSuperview(inset: 16, relation: .equal)
+            $0.top.pinToSuperviewMargin(inset: 16, relation: .equal)
+        }
+        
+        self.zoomButtonContainer.layer.cornerRadius = 4
+        self.zoomButtonContainer.layer.masksToBounds = true
+        self.zoomButtonContainer.backgroundColor = UIColor(red: 50, green: 50, blue: 50, alpha: 1)
```

Comment:
What’s the point of using a blur view below if this has a solid background color?